identity/src/ecdsa.rs

```diff
@@ -34,7 +34,7 @@ use p256::{
 use void::Void;
 
 /// An ECDSA keypair.
-#[derive(Clone)]
+#[derive(Clone, PartialEq, Eq)]
```

Comment:
Secret keys should not implement regular `PartialEq` because those implementations are not constant-time and can be used to leak the keys. Please remove these again.